lib/prefatory/config.rb

```diff
@@ -16,6 +16,7 @@ module Prefatory
     setting :primary_uuid, :primary_uuid
   end
   setting :storage do
+    setting :redis_client
```

Comment:
imo, there shouldn't be any provider-specific options exposed at this level to users of the gem. but, I think addressing this would need some design/discussion and would end up being a PR of a much larger scope... so maybe something to consider later on if we continue to use this gem?

Reply:
I thought about that a bit too. In sidekiq for example, you pass a REDIS_URL and it manages its own connection pool. We could do that here too, but that would mean that this gem might need its own Redis when scaling up. 
Passing an object allows us to reuse the same connection pool we use across the entire app. 

I've seen other examples of provider-specific configuration exposed at that same level (Carrierwave comes to my mind), but maybe the gem config needs to be re-adjusted a bit. But I agree that this is probably part of a larger discussion.